Configure.cmake

```diff
@@ -54,6 +54,7 @@ set(SLEEF_SUPPORTED_EXTENSIONS
   ADVSIMD ADVSIMDNOFMA SVE SVENOFMA                     # Aarch64
   NEON32 NEON32VFPV4                                    # Aarch32
   VSX VSXNOFMA                                          # PPC64
+  Z13 Z13NOFMA Z14 Z14NOFMA		                # IBM Z
```

Comment:
Would it be better if `ZVECTOR` and `ZVECTOR2` are used instead of CPU names?

Reply:
Done.